pkgs/games/openra_2019/default.nix

```diff
@@ -0,0 +1,71 @@
+/*  This file defines all OpenRA packages under `openraPackages`,
+    e.g. the OpenRA release engine can be found at `openraPackages.engines.release` (see `engines.nix`),
+    or the out-of-tree mod "Combined Arms" can be found at `openraPackages.mods.ca` (see `mods.nix`).
+    The `openra` package is just an alias to `openraPackages.engines.release`,
+    and just provides the mods included in the source code of the engine.
+    Additional engines or mods can be added with `openraPackages.buildOpenRAEngine` (function around `engine.nix`)
+    and `openraPackages.buildOpenRAMod` (function around `mod.nix`), respectively.
+*/
+pkgs:
+
+with pkgs.lib;
```

Comment:
pkgs should not be used as an input and inside of nixpkgs pkgs.lib should also not be used

Reply:
Yes, sure but it would still be cool if you could improve the old sections a little, too.